### ISSUE TITLE
Add ability to temporarily display sitewide notices

### ISSUE THREAD

**Issue by jvendetti** [ncbo/bioportal_web_ui#33]
Occasionally it would be helpful to display sitewide notices, for example:

"Advanced ontology upload processing will be suspended until Sunday 2300 (GMT-8) for routine system maintenance."

... during periods when we have to turn off cron to reparse all ontologies after UMLS imports.

**Comment by jvendetti** [comment]
There's some existing code already present that supports this. It involves assigning the desired notice to the [$SITE_NOTICE variable](https://github.com/ncbo/bioportal_config/blob/master/bioportal_web_ui/config/bioportal_config_production.rb#L195) in the [environment specific configuration file](https://github.com/ncbo/bioportal_web_ui/blob/master/config/environments/production.rb#L84).

Looks something like the following (using test verbiage):

![Screenshot 2020-03-26 16 51 25](https://user-images.githubusercontent.com/1696923/77707349-18291a80-6f82-11ea-87b8-eb11172e9688.png)

**Comment by alexskr** [comment]
Issuing site-wide notices using a $SITE_NOTICE config setting requires updating config files and performing deployment of the UI which is a bit heavy-handed and requires operations or development team to do it.   It is great that we have this option but It would be even better to be able to set those notices in the UI Admin or API especially for the appliance users.

**Comment by jvendetti** [comment]
I agree that it's not an ideal solution, which is why I didn't mark the issue as closed. The purpose of the comment was to document some code that I didn't know existed until yesterday. This code could be leveraged until such time that we're able to implement a better solution.

**Comment by graybeal** [comment]
Yes, the emphasis of this task is on a temporary solution, that could flip that display on and off with a simple admin (or even Ruby) command. 

But why can't $SITE_NOTICE be set with a Ruby command, and take effect immediately? (I am aware that bad HTML could cause all sorts of UI trouble. But there are ways to mitigate that risk.)
